src/ReactiveDomain/ReactiveDomain.Tests/Logging/with_message_logging_disabled.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Threading;
+using ReactiveDomain.Logging;
+
+namespace ReactiveDomain.Tests.Logging
+{
+    // ReSharper disable once InconsistentNaming
+    public abstract class with_message_logging_disabled :
+        with_event_store,
+        IDisposable
+    {
+        //protected member Logging class that inherits from QueuedSubscriber
+        protected EventStoreMessageLogger Logging;
+        protected string StreamName = $"Test-{Guid.NewGuid():N}-";
```

Comment:
Is the trailing dash on StreamName intentional?

Reply:
Yes. Test-guidxxxxxxx-Log-date. It was hard to read with the guid running into the word Log.

Reply:
or maybe not. Looks like I've added the hyphen twice.